### PR TITLE
Fix/zscaler destination ip

### PR DESCRIPTION
**Pull request by maximehgn**
add destination.ip for zscaler web events

### CODE REVIEW

File: Zscaler/zscaler-zia/ingest/parser.yml [hunk 1]
```diff
@@ -133,9 +133,8 @@ stages:
...
 
       - set:
-          destination.ip: "{{json_event.message.event.srv_dip or json_event.message.event.sdip or json_event.message.event.destinationip}}"
-        filter: "{{(json_event.message.event.srv_dip != null and json_event.message.event.srv_dip | is_ipaddress) or (json_event.message.event.sdip != null and json_event.message.event.sdip | is_ipaddress) or (json_event.message.event.destinationip != null and json_event.message.event.destinationip | is_ipaddress)}}"
-
+          destination.ip: "{{json_event.message.event.srv_dip or json_event.message.event.sdip or json_event.message.event.destinationip or json_event.message.event.serverip}}"
+        filter: "{{(json_event.message.event.srv_dip != null and json_event.message.event.srv_dip | is_ipaddress) or (json_event.message.event.sdip != null and json_event.message.event.sdip | is_ipaddress) or (json_event.message.event.destinationip != null and json_event.message.event.destinationip | is_ipaddress) or (json_event.message.event.serverip != null and json_event.message.event.serverip | is_ipaddress)}}"
```
Comment (PierrickV) [inline Zscaler/zscaler-zia/ingest/parser.yml:137]:
```suggestion
        filter: "{{json_event.message.event.srv_dip | is_ipaddress
  or json_event.message.event.sdip | is_ipaddress
  or json_event.message.event.destinationip | is_ipaddress
  or json_event.message.event.serverip | is_ipaddress }}"
```

Comment (PierrickV) [reply Zscaler/zscaler-zia/ingest/parser.yml:137]: `is_ipaddress` already checks for none

Comment (maximehgn) [reply Zscaler/zscaler-zia/ingest/parser.yml:137]: I just based my code on what @squioc done before, didn't know but thanks!

Comment (maximehgn) [reply Zscaler/zscaler-zia/ingest/parser.yml]: @PierrickV the YAML test didn't pass your changes, I wanted to regroup your changes on 1 unique line but I had another test failed (intake format one) and I admit that I don't understand why

**Review by Adamowoc — APPROVED** [review]

File: Zscaler/zscaler-zia/ingest/parser.yml (hunk 1 above)
Comment (PierrickV) [reply Zscaler/zscaler-zia/ingest/parser.yml]: There was an orphan parenthesis at the beging of your code
```
filter: "{{(
-----------^
```

**Comment by PierrickV** [comment]
It should work now @maximehgn =)

**Review by PierrickV — APPROVED** [review]

**Pull request merged**